libs/openFrameworks/graphics/ofPixels.cpp

```diff
@@ -443,18 +443,18 @@ const PixelType * ofPixels_<PixelType>::getData() const{
 }
 
 template<typename PixelType>
-void ofPixels_<PixelType>::allocate(int w, int h, int _channels){
+void ofPixels_<PixelType>::allocate(size_t w, size_t h, size_t _channels){
 	allocate(w,h,pixelFormatFromNumChannels(_channels));
 }
 
 template<typename PixelType>
-void ofPixels_<PixelType>::allocate(int w, int h, ofPixelFormat format){
+void ofPixels_<PixelType>::allocate(size_t w, size_t h, ofPixelFormat format){
 	if (w <= 0 || h <= 0 || format==OF_PIXELS_UNKNOWN) {
```

Comment:
Probably change to == 0 on w/ h, as some compilers may give warnings since w/h can now never be < 0.